cloud/helm-chart/src/main/helm/artifacthub-repo.yml

```diff
@@ -0,0 +1,6 @@
+# This file is used by ArtifactHub to verify the repository.
```

Comment:
I think @jfallows has already tried this approach and it did not work. The [documentation](https://artifacthub.io/docs/topics/repositories/#verified-publisher) says you have to put this file to the _repository_ root (not inside a helm chart). It mentions http and git repositories specifically, but there is no info on whether oci repositories are supported.

Reply:
Well that doesn't sound promising. Maybe adding the 'source' link in the chart will help? I'll do some more digging to double check.

Reply:
I found this repo that is verified and has the same setup: https://github.com/coder/coder/blob/main/helm/Chart.yaml https://artifacthub.io/packages/helm/coder-v2/coder
Hopefully that means it will work this time.

Reply:
I see so the deploy operation will need to either generate this file or deploy this file along with the chart? Here is how Chartmuseum adds this https://github.com/helm/chartmuseum#artifact-hub

Reply:
We decided to go with ghcr's oci based helm repository. I am not sure if you can just push arbitrary files (e.g. a yaml file) to the oci repo. Also there is no word in the artifacthub doc about how you can get an oci based repo verified. Chartmuseum implements a http (rest) based helm repo, so that's not an answer to our question.

Reply:
looks like we can push it seprately: https://artifacthub.io/docs/topics/repositories/helm-charts/#oci-support